index.md

```diff
@@ -43,35 +43,9 @@ as well.  And it isn't just for software: books,
 papers, small data sets, and anything that changes over time or needs
 to be shared can and should be stored in a version control system.
 
-> ## Prerequisites {.prereq}
+> ## Prerequisites
 >
 > In this lesson we use Git from the Unix Shell.
 > Some previous experience with the shell is expected,
 > *but isn't mandatory*.
-
-> ## Getting ready {.getready}
```

Comment:
Is the "Getting ready" callout going somewhere else? I think we should keep (in this way or another) to be sure everybody is starting from the same setup.